packages/cashier/src/constants/transaction-status.tsx

```diff
@@ -87,6 +87,20 @@ export const getStatus = (
                 renderer: 'unsuccessful',
                 transaction_hash: localize('NA'),
             },
+            REVERTED: {
```

Comment:
@nada-deriv Need to also update `packages/cashier/src/modules/crypto-transactions/helpers/crypto-transactions-mapper.tsx` 🙏🏻